client/scripts/views/pages/members.ts

```diff
@@ -11,14 +12,40 @@ import PageLoading from 'views/pages/loading';
 import User from 'views/components/widgets/user';
 import Sublayout from 'views/sublayout';
 import { CommunityOptionsPopover } from './discussions';
+import { ConsoleLoggerImpl } from 'typescript-logging';
+import { Profile } from 'client/scripts/models';
+import address from 'server/models/address';
+import { min } from 'underscore';
+import ConfirmSnapshotVoteModal from '../modals/confirm_snapshot_vote_modal';
+
+// The number of member profiles that are batch loaded
+const DEFAULT_MEMBER_SIZE = 20;
```

Comment:
Let's make this `DEFAULT_MEMBER_REQ_SIZE` so that it's ever so slightly clearer. and then you don't need the comment above it.